twitter_post_block.py

```diff
@@ -69,9 +63,12 @@ def _post_tweet(self, payload):
 
         status = response.status_code
         if status != 200:
+            response = response.json()
```

Comment:
I find that it's after to wrap these in a try block. You never know that it's going to be json every time.